app/controllers/v1/hackbot/webhook_controller.rb

```diff
@@ -8,8 +8,10 @@ def callback
         elsif params[:type] != 'event_callback'
           render status: :not_implemented
         else
-          handle_message
           render status: 200
+
```

Comment:
OK, I realise I haven't been super clear here.

Basically, I'm struggling with fixing #82.

Using threading as I did here was a bad bad bad solution, because Rails is not thread safe.

I'd appreciate some help coming up with a solution for this asynchronous tasks.

I'm thinking it might make sense to have a job here?